### PR TITLE
Vpc with ec2

### PR DESCRIPTION
**Pull request by jamesdehart**
Let me know what you think @tcondit

### CODE REVIEW

File: infrastructure/vpc-with-ec2/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    * S3 VPC Endpoint
+  * default rotue table
+* Security Group
+  * HTTP and SSH ingress
+* S3
+  * private acl
+  * Server side encryption
+    * KMS
+* KMS
+* EC2 Intance
```
Comment (tcondit) [inline infrastructure/vpc-with-ec2/README.md]: `s/Intance/Instance/`

File: infrastructure/vpc-with-ec2/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| aws\_region | The AWS region. | string | `"us-west-2"` | no |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| default\_route\_id | VPC's default routetable id. |
```
Comment (tcondit) [inline infrastructure/vpc-with-ec2/README.md]: More of the same (nitpicking) `s/id/ID/`

File: infrastructure/vpc-with-ec2/ec2.tf [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+}
+
+#
+# EC2 Pem file.
+#
+resource "aws_key_pair" "zombie" {
```
Comment (tcondit) [inline infrastructure/vpc-with-ec2/ec2.tf]: Just curious why this is called "zombie".

File: infrastructure/vpc-with-ec2/s3.tf [hunk 4]
```diff
@@ -0,0 +1,32 @@
+#
+# s3.tf - Property of Skillfox Labs LLC
+#
+# Creates an S3 Bucket
+#  - server side encryption with KMS.
+#  - blocks public policys & access control lists.
```
Comment (tcondit) [inline infrastructure/vpc-with-ec2/s3.tf]: `s/policys/policies/`

**Comment by tcondit** [comment]
Ping! So far there are no replies and no merge.

File: README.md [hunk 5]
```diff
@@ -1 +1,27 @@
...
+This repo contains infrastructure as code for AWS.
+
+
+## Ready ...
+
+Whether you try that project or not, please make sure you've got everything on this list:
+* This repo.
+* An AWS account. ([sign up](https://aws.amazon.com/))
+* HashiCorp has now provided a training site to [install](https://learn.hashicorp.com/terraform/getting-started/install.html).
+* AWS credentials ([configure](https://pulumi.io/quickstart/aws/setup.html))
```
[the remark this answers is not in the document]
Comment (jamesdehart) [reply README.md]: Why update the url? The content that is provided is good.

File: infrastructure/vpc-with-ec2/README.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+  * /16
+* IGW
+* Subnet
+  * Public subnet
+  * /20
+* Route Table
+  * Public Route table
+    * Internet gateway -> 0.0.0.0/0
+    * S3 VPC Endpoint
+  * default rotue table
```
[the remark this answers is not in the document]
Comment (jamesdehart) [reply infrastructure/vpc-with-ec2/README.md]: fixed

File: infrastructure/vpc-with-ec2/README.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+* Security Group
+  * HTTP and SSH ingress
+* S3
+  * private acl
```
[the remark this answers is not in the document]
Comment (jamesdehart) [reply infrastructure/vpc-with-ec2/README.md]: Fixed. Good point.

File: infrastructure/vpc-with-ec2/README.md (hunk 1 above)
Comment (jamesdehart) [reply infrastructure/vpc-with-ec2/README.md]: Fixed.

File: infrastructure/vpc-with-ec2/README.md (hunk 2 above)
Comment (jamesdehart) [reply infrastructure/vpc-with-ec2/README.md]: fixed.

File: infrastructure/vpc-with-ec2/ec2.tf (hunk 3 above)
Comment (jamesdehart) [reply infrastructure/vpc-with-ec2/ec2.tf]: it was from my computer named zombie.

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (tcondit) [reply README.md]: It's been 19 days. I forget why. Next time I'll add more details.

File: infrastructure/vpc-with-ec2/ec2.tf (hunk 3 above)
Comment (tcondit) [reply infrastructure/vpc-with-ec2/ec2.tf]: LOL

**Pull request merged**